hello-thirdparty/README.md

```diff
@@ -5,13 +5,16 @@ Hello Third Party Lib is an Android C++ sample that depends on a prebuilt static
 Pre-requisites
 --------------
 - Android Studio 1.3+ with [NDK](https://developer.android.com/ndk/) bundle.
-- [Play Games C++ SDK](https://developers.google.com/games/services/downloads/sdks)
+
+Project structure
+-----------------
+This project consists of 2 sub-projects.  The first the "app" and contains
+the application code and resources.  It is dependendent on the second sub-project
+"gpg-sdk" which downloads and unzips the Google Play Games SDK.
```

Comment:
what about naming the directory `thirdparty`? that way it would be more obvious it's not related to the code.